Core/src/org/sleuthkit/autopsy/casemodule/NewCaseWizardAction.java

```diff
@@ -75,11 +79,36 @@ private void runNewCaseWizard() {
                 @Override
                 protected Void doInBackground() throws Exception {
                     String caseNumber = (String) wizardDescriptor.getProperty("caseNumber"); //NON-NLS
-                    String examiner = (String) wizardDescriptor.getProperty("caseExaminer"); //NON-NLS
+                    String examinerName = (String) wizardDescriptor.getProperty("caseExaminerName"); //NON-NLS
+                    String examinerPhone = (String) wizardDescriptor.getProperty("caseExaminerPhone"); //NON-NLS
+                    String examinerEmail = (String) wizardDescriptor.getProperty("caseExaminerEmail"); //NON-NLS
+                    String caseNotes = (String) wizardDescriptor.getProperty("caseNotes"); //NON-NLS
+                    String organizationName = (String) wizardDescriptor.getProperty("caseOrganization"); //NON-NLS
                     final String caseName = (String) wizardDescriptor.getProperty("caseName"); //NON-NLS
                     String createdDirectory = (String) wizardDescriptor.getProperty("createdDirectory"); //NON-NLS
                     CaseType caseType = CaseType.values()[(int) wizardDescriptor.getProperty("caseType")]; //NON-NLS
-                    Case.createAsCurrentCase(createdDirectory, caseName, caseNumber, examiner, caseType);
+                    Case.createAsCurrentCase(createdDirectory, new CaseDetails(caseName, caseNumber, examinerName, examinerPhone, examinerEmail, caseNotes), caseType);
+                    if (EamDb.isEnabled()) {  //if the eam is enabled we need to save the case organization information now
+                        try {
+                            EamDb dbManager = EamDb.getInstance();
+                            if (dbManager != null) {
+                                CorrelationCase cRCase = dbManager.getCaseByUUID(Case.getCurrentCase().getName());
+                                if (cRCase == null) {
+                                    cRCase = dbManager.newCase(Case.getCurrentCase());
+                                }
+                                if (!organizationName.isEmpty()) {
+                                    for (EamOrganization org : dbManager.getOrganizations()) {
+                                        if (org.getName().equals(organizationName)) {
+                                            cRCase.setOrg(org);
+                                            dbManager.updateCase(cRCase);
+                                        }
+                                    }
+                                }
+                            }
+                        } catch (EamDbException ex) {
+
```

Comment:
We should only rarely swallow exceptions.  Here, we should allow the exception to propagate and be handled in the catch block for the get() call in the done() method, with the reasoning that if the user has decided to use the central repository, they should be notified if something goes wrong creating the case in the CR.